prow/flagutil/github.go

```diff
@@ -175,17 +171,3 @@ func (o *GitHubOptions) GitClient(secretAgent *secret.Agent, dryRun bool) (clien
 
 	return client, nil
 }
-
-// GitHubOAuthClient returns an oauth client.
-func (o *GitHubOptions) GitHubOAuthClient(oauthConfig *oauth2.Config) githuboauth.OAuthClient {
-	oauthConfig.Endpoint = oauth2.Endpoint{
-		AuthURL:  fmt.Sprintf("https://%s/login/oauth/authorize", o.host),
-		TokenURL: fmt.Sprintf("https://%s/login/oauth/access_token", o.host),
-	}
-	return githuboauth.NewClient(oauthConfig)
-}
-
-// GetGitHubClient returns a github client wrapper.
-func (o *GitHubOptions) GetGitHubClient(accessToken string, dryRun bool) githuboauth.GitHubClientWrapper {
-	return ghclient.NewClientWithEndpoint(o.endpoint.Strings()[0], accessToken, dryRun)
```

Comment:
You might ask -- is it even valid to use `prow/github` if we're routing through `ghproxy` for these calls to get the ID of the authenticated user? But we did in the past, as well!